checker/manual/subtyping-checker.tex

```diff
@@ -36,32 +36,53 @@ \section{Using the Subtyping Checker\label{subtyping-using}}
 The Subtyping Checker is used in the same way as other checkers (using the
 \code{-processor org.checkerframework.common.subtyping.SubtypingChecker} option; see Chapter~\ref{using-a-checker}), except that it
 requires an additional annotation processor argument via the standard
-``\code{-A}'' switch:
+``\code{-A}'' switch. One of the two following arguments must be used with the
+Subtyping Checker:
 \end{sloppypar}
 
 \begin{itemize}
 
 \item
-\code{-Aquals}: this option specifies a comma-no-space-separated list of
-the fully-qualified class
-names of the annotations used as qualifiers in the custom type system.  For
-example,
+Provide the fully-qualified class name(s) of the annotation(s) in the custom
+type system through the \code{-Aquals} option, using a comma-no-space-separated
+notation:
 
 \begin{alltt}
-  javac -processor org.checkerframework.common.subtyping.SubtypingChecker
-        \textit{-Aquals=myproject.qual.MyQual,myproject.qual.OtherQual} MyFile.java ...
+  javac -Xbootclasspath/p:\textit{/full/path/to/myProject/bin}:\textit{/full/path/to/myLibrary/bin} \ttbs
+        -processor org.checkerframework.common.subtyping.SubtypingChecker \ttbs
+        -Aquals=\textit{myModule.qual.MyQual},\textit{myModule.qual.OtherQual} MyFile.java ...
 \end{alltt}
 
-It serves the same purpose as the \refqualclass{framework/qual}{TypeQualifiers}
-annotation used by other checkers (see section
-\ref{writing-compiler-interface}).
-
 The annotations listed in \code{-Aquals} must be accessible to
 the compiler during compilation in the classpath.  In other words, they must
 already be compiled (and, typically, be on the javac bootclasspath)
 before you run the Subtyping Checker with \code{javac}.  It
 is not sufficient to supply their source files on the command line.
 
+\item
+Provide the fully-qualified paths to a set of directories that contain the
```

Comment:
Similar to the fenum checker, this will be added as an example project under examples/subtyping-extension with similar test setup.